2022-19-07 01:31:54[INFO][restartedMain][c.b.a.AddressBookAppApplication:55] Starting AddressBookAppApplication using Java 18.0.1.1 on LAPTOP-RR74P28L with PID 21192 (C:\Users\Admin\Documents\BridgelabzCFP\AddressBookApp\target\classes started by root in C:\Users\Admin\Documents\BridgelabzCFP\AddressBookApp)
2022-19-07 01:31:54[INFO][restartedMain][c.b.a.AddressBookAppApplication:640] The following 1 profile is active: "dev"
2022-19-07 01:31:54[INFO][restartedMain][o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor:255] Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2022-19-07 01:31:54[INFO][restartedMain][o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor:255] For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2022-19-07 01:31:54[INFO][restartedMain][o.s.d.r.c.RepositoryConfigurationDelegate:132] Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2022-19-07 01:31:54[INFO][restartedMain][o.s.d.r.c.RepositoryConfigurationDelegate:201] Finished Spring Data repository scanning in 45 ms. Found 1 JPA repository interfaces.
2022-19-07 01:31:55[INFO][restartedMain][o.s.b.w.e.t.TomcatWebServer:108] Tomcat initialized with port(s): 8080 (http)
2022-19-07 01:31:55[INFO][restartedMain][o.a.c.c.StandardService:173] Starting service [Tomcat]
2022-19-07 01:31:55[INFO][restartedMain][o.a.c.c.StandardEngine:173] Starting Servlet engine: [Apache Tomcat/9.0.64]
2022-19-07 01:31:55[INFO][restartedMain][o.a.c.c.C.[.[.[/]:173] Initializing Spring embedded WebApplicationContext
2022-19-07 01:31:55[INFO][restartedMain][o.s.b.w.s.c.ServletWebServerApplicationContext:292] Root WebApplicationContext: initialization completed in 1400 ms
2022-19-07 01:31:55[INFO][restartedMain][o.h.j.i.u.LogHelper:31] HHH000204: Processing PersistenceUnitInfo [name: default]
2022-19-07 01:31:55[INFO][restartedMain][o.h.Version:44] HHH000412: Hibernate ORM core version 5.6.9.Final
2022-19-07 01:31:55[INFO][restartedMain][o.h.a.c.Version:56] HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-19-07 01:31:56[INFO][restartedMain][c.z.h.HikariDataSource:110] HikariPool-1 - Starting...
2022-19-07 01:31:56[INFO][restartedMain][c.z.h.HikariDataSource:123] HikariPool-1 - Start completed.
2022-19-07 01:31:56[INFO][restartedMain][o.h.d.Dialect:175] HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-19-07 01:31:56[INFO][restartedMain][o.h.e.t.j.p.i.JtaPlatformInitiator:52] HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-19-07 01:31:56[INFO][restartedMain][o.s.o.j.LocalContainerEntityManagerFactoryBean:437] Initialized JPA EntityManagerFactory for persistence unit 'default'
2022-19-07 01:31:57[WARN][restartedMain][o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration:219] spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2022-19-07 01:31:57[INFO][restartedMain][o.s.b.d.a.OptionalLiveReloadServer:58] LiveReload server is running on port 35729
2022-19-07 01:31:57[INFO][restartedMain][o.s.b.w.e.t.TomcatWebServer:220] Tomcat started on port(s): 8080 (http) with context path ''
2022-19-07 01:31:57[INFO][restartedMain][c.b.a.AddressBookAppApplication:61] Started AddressBookAppApplication in 3.651 seconds (JVM running for 4.335)
2022-19-07 01:31:57[INFO][restartedMain][c.b.a.AddressBookAppApplication:12] Welcome to Address Book App!
2022-19-07 01:32:06[INFO][http-nio-8080-exec-1][o.a.c.c.C.[.[.[/]:173] Initializing Spring DispatcherServlet 'dispatcherServlet'
2022-19-07 01:32:06[INFO][http-nio-8080-exec-1][o.s.w.s.DispatcherServlet:525] Initializing Servlet 'dispatcherServlet'
2022-19-07 01:32:06[INFO][http-nio-8080-exec-1][o.s.w.s.DispatcherServlet:547] Completed initialization in 0 ms
2022-19-07 01:32:43[WARN][http-nio-8080-exec-2][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [com.bridgelabz.addressbookapp.exceptions.AddressBookException: Address Book with id 2 not found!]
2022-19-07 01:32:49[WARN][http-nio-8080-exec-10][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [com.bridgelabz.addressbookapp.exceptions.AddressBookException: Address Book with id 1 not found!]
2022-19-07 01:33:17[WARN][http-nio-8080-exec-9][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [1] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.editAddressBook(int,com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 3 errors: [Field error in object 'addressBookDTO' on field 'address': rejected value [null]; codes [NotBlank.addressBookDTO.address,NotBlank.address,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.address,address]; arguments []; default message [address]]; default message [Address should not be empty]] [Field error in object 'addressBookDTO' on field 'city': rejected value [null]; codes [NotBlank.addressBookDTO.city,NotBlank.city,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.city,city]; arguments []; default message [city]]; default message [City should not be empty]] [Field error in object 'addressBookDTO' on field 'state': rejected value [null]; codes [NotBlank.addressBookDTO.state,NotBlank.state,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.state,state]; arguments []; default message [state]]; default message [State should not be empty]] ]
2022-19-07 01:33:36[WARN][http-nio-8080-exec-4][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [com.bridgelabz.addressbookapp.exceptions.AddressBookException: Address Book with id 1 not found!]
2022-19-07 01:36:02[WARN][http-nio-8080-exec-3][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 4 errors: [Field error in object 'addressBookDTO' on field 'firstName': rejected value [kabir]; codes [Pattern.addressBookDTO.firstName,Pattern.firstName,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.firstName,firstName]; arguments []; default message [firstName],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[A-Z][a-zA-Z]{2,}$]; default message [First name invalid]] [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [profile_02]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] [Field error in object 'addressBookDTO' on field 'zipcode': rejected value [all in one]; codes [Pattern.addressBookDTO.zipcode,Pattern.zipcode,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.zipcode,zipcode]; arguments []; default message [zipcode],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[1-9]\d{2}\s?\d{3}$]; default message [Zip code invalid]] ]
2022-19-07 01:36:37[WARN][http-nio-8080-exec-5][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 5 errors: [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [profile_02]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] [Field error in object 'addressBookDTO' on field 'zipcode': rejected value [all in one]; codes [Pattern.addressBookDTO.zipcode,Pattern.zipcode,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.zipcode,zipcode]; arguments []; default message [zipcode],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[1-9]\d{2}\s?\d{3}$]; default message [Zip code invalid]] [Field error in object 'addressBookDTO' on field 'firstName': rejected value [kabir]; codes [Pattern.addressBookDTO.firstName,Pattern.firstName,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.firstName,firstName]; arguments []; default message [firstName],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[A-Z][a-zA-Z]{2,}$]; default message [First name invalid]] [Field error in object 'addressBookDTO' on field 'lastname': rejected value [nanda]; codes [Pattern.addressBookDTO.lastname,Pattern.lastname,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.lastname,lastname]; arguments []; default message [lastname],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[A-Z][a-zA-Z]{2,}$]; default message [Last name invalid]] ]
2022-19-07 01:36:57[WARN][http-nio-8080-exec-7][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 3 errors: [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [profile_02]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'zipcode': rejected value [all in one]; codes [Pattern.addressBookDTO.zipcode,Pattern.zipcode,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.zipcode,zipcode]; arguments []; default message [zipcode],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[1-9]\d{2}\s?\d{3}$]; default message [Zip code invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:37:10[WARN][http-nio-8080-exec-9][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 3 errors: [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [profile_02]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'zipcode': rejected value [390  098]; codes [Pattern.addressBookDTO.zipcode,Pattern.zipcode,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.zipcode,zipcode]; arguments []; default message [zipcode],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[1-9]\d{2}\s?\d{3}$]; default message [Zip code invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:37:17[WARN][http-nio-8080-exec-4][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 2 errors: [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [profile_02]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:37:22[WARN][http-nio-8080-exec-3][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 2 errors: [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [profile_02]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:37:29[WARN][http-nio-8080-exec-5][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 3 errors: [Field error in object 'addressBookDTO' on field 'zipcode': rejected value [3900 98]; codes [Pattern.addressBookDTO.zipcode,Pattern.zipcode,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.zipcode,zipcode]; arguments []; default message [zipcode],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^[1-9]\d{2}\s?\d{3}$]; default message [Zip code invalid]] [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [profile_02]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:37:52[WARN][http-nio-8080-exec-7][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO) with 2 errors: [Field error in object 'addressBookDTO' on field 'phoneNo': rejected value [01 90909090999]; codes [Pattern.addressBookDTO.phoneNo,Pattern.phoneNo,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.phoneNo,phoneNo]; arguments []; default message [phoneNo],[Ljavax.validation.constraints.Pattern$Flag;@216baf4b,^(\d{0,2}\s?)*[1-9]\d{9}$]; default message [Phone No. invalid]] [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:37:57[WARN][http-nio-8080-exec-8][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:38:09[WARN][http-nio-8080-exec-4][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:38:17[WARN][http-nio-8080-exec-3][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'email': rejected value [km.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@7e1a2b09,.*]; default message [Email is invalid]] ]
2022-19-07 01:40:22[INFO][SpringApplicationShutdownHook][o.s.o.j.LocalContainerEntityManagerFactoryBean:651] Closing JPA EntityManagerFactory for persistence unit 'default'
2022-19-07 01:40:23[INFO][SpringApplicationShutdownHook][c.z.h.HikariDataSource:350] HikariPool-1 - Shutdown initiated...
2022-19-07 01:40:23[INFO][SpringApplicationShutdownHook][c.z.h.HikariDataSource:352] HikariPool-1 - Shutdown completed.
2022-19-07 01:40:33[INFO][restartedMain][c.b.a.AddressBookAppApplication:55] Starting AddressBookAppApplication using Java 18.0.1.1 on LAPTOP-RR74P28L with PID 20128 (C:\Users\Admin\Documents\BridgelabzCFP\AddressBookApp\target\classes started by root in C:\Users\Admin\Documents\BridgelabzCFP\AddressBookApp)
2022-19-07 01:40:33[INFO][restartedMain][c.b.a.AddressBookAppApplication:640] The following 1 profile is active: "dev"
2022-19-07 01:40:33[INFO][restartedMain][o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor:255] Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2022-19-07 01:40:33[INFO][restartedMain][o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor:255] For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2022-19-07 01:40:33[INFO][restartedMain][o.s.d.r.c.RepositoryConfigurationDelegate:132] Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2022-19-07 01:40:33[INFO][restartedMain][o.s.d.r.c.RepositoryConfigurationDelegate:201] Finished Spring Data repository scanning in 45 ms. Found 1 JPA repository interfaces.
2022-19-07 01:40:34[INFO][restartedMain][o.s.b.w.e.t.TomcatWebServer:108] Tomcat initialized with port(s): 8080 (http)
2022-19-07 01:40:34[INFO][restartedMain][o.a.c.c.StandardService:173] Starting service [Tomcat]
2022-19-07 01:40:34[INFO][restartedMain][o.a.c.c.StandardEngine:173] Starting Servlet engine: [Apache Tomcat/9.0.64]
2022-19-07 01:40:34[INFO][restartedMain][o.a.c.c.C.[.[.[/]:173] Initializing Spring embedded WebApplicationContext
2022-19-07 01:40:34[INFO][restartedMain][o.s.b.w.s.c.ServletWebServerApplicationContext:292] Root WebApplicationContext: initialization completed in 1350 ms
2022-19-07 01:40:34[INFO][restartedMain][o.h.j.i.u.LogHelper:31] HHH000204: Processing PersistenceUnitInfo [name: default]
2022-19-07 01:40:34[INFO][restartedMain][o.h.Version:44] HHH000412: Hibernate ORM core version 5.6.9.Final
2022-19-07 01:40:34[INFO][restartedMain][o.h.a.c.Version:56] HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-19-07 01:40:34[INFO][restartedMain][c.z.h.HikariDataSource:110] HikariPool-1 - Starting...
2022-19-07 01:40:35[INFO][restartedMain][c.z.h.HikariDataSource:123] HikariPool-1 - Start completed.
2022-19-07 01:40:35[INFO][restartedMain][o.h.d.Dialect:175] HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-19-07 01:40:35[INFO][restartedMain][o.h.e.t.j.p.i.JtaPlatformInitiator:52] HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-19-07 01:40:35[INFO][restartedMain][o.s.o.j.LocalContainerEntityManagerFactoryBean:437] Initialized JPA EntityManagerFactory for persistence unit 'default'
2022-19-07 01:40:35[WARN][restartedMain][o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration:219] spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2022-19-07 01:40:36[INFO][restartedMain][o.s.b.d.a.OptionalLiveReloadServer:58] LiveReload server is running on port 35729
2022-19-07 01:40:36[INFO][restartedMain][o.s.b.w.e.t.TomcatWebServer:220] Tomcat started on port(s): 8080 (http) with context path ''
2022-19-07 01:40:36[INFO][restartedMain][c.b.a.AddressBookAppApplication:61] Started AddressBookAppApplication in 3.573 seconds (JVM running for 4.295)
2022-19-07 01:40:36[INFO][restartedMain][c.b.a.AddressBookAppApplication:12] Welcome to Address Book App!
2022-19-07 01:41:44[INFO][http-nio-8080-exec-2][o.a.c.c.C.[.[.[/]:173] Initializing Spring DispatcherServlet 'dispatcherServlet'
2022-19-07 01:41:44[INFO][http-nio-8080-exec-2][o.s.w.s.DispatcherServlet:525] Initializing Servlet 'dispatcherServlet'
2022-19-07 01:41:44[INFO][http-nio-8080-exec-2][o.s.w.s.DispatcherServlet:547] Completed initialization in 10 ms
2022-19-07 01:41:44[WARN][http-nio-8080-exec-2][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'address': rejected value []; codes [NotBlank.addressBookDTO.address,NotBlank.address,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.address,address]; arguments []; default message [address]]; default message [Address should not be empty]] ]
2022-19-07 01:41:52[WARN][http-nio-8080-exec-5][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'city': rejected value []; codes [NotBlank.addressBookDTO.city,NotBlank.city,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.city,city]; arguments []; default message [city]]; default message [City should not be empty]] ]
2022-19-07 01:41:58[WARN][http-nio-8080-exec-6][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'state': rejected value []; codes [NotBlank.addressBookDTO.state,NotBlank.state,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.state,state]; arguments []; default message [state]]; default message [State should not be empty]] ]
2022-19-07 01:42:12[WARN][http-nio-8080-exec-8][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'email': rejected value [km@]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2a5483ad,.*]; default message [Email is invalid]] ]
2022-19-07 01:42:20[WARN][http-nio-8080-exec-10][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'email': rejected value [km@a.]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2a5483ad,.*]; default message [Email is invalid]] ]
2022-19-07 01:42:47[WARN][http-nio-8080-exec-2][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'email': rejected value [k.@gmail.com]; codes [Email.addressBookDTO.email,Email.email,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2a5483ad,.*]; default message [Email is invalid]] ]
2022-19-07 01:43:25[WARN][http-nio-8080-exec-4][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [com.bridgelabz.addressbookapp.exceptions.AddressBookException: Address Book with id 2 not found!]
2022-19-07 01:45:16[INFO][SpringApplicationShutdownHook][o.s.o.j.LocalContainerEntityManagerFactoryBean:651] Closing JPA EntityManagerFactory for persistence unit 'default'
2022-19-07 01:45:16[INFO][SpringApplicationShutdownHook][c.z.h.HikariDataSource:350] HikariPool-1 - Shutdown initiated...
2022-19-07 01:45:16[INFO][SpringApplicationShutdownHook][c.z.h.HikariDataSource:352] HikariPool-1 - Shutdown completed.
2022-19-07 01:45:27[INFO][restartedMain][c.b.a.AddressBookAppApplication:55] Starting AddressBookAppApplication using Java 18.0.1.1 on LAPTOP-RR74P28L with PID 17172 (C:\Users\Admin\Documents\BridgelabzCFP\AddressBookApp\target\classes started by root in C:\Users\Admin\Documents\BridgelabzCFP\AddressBookApp)
2022-19-07 01:45:27[INFO][restartedMain][c.b.a.AddressBookAppApplication:640] The following 1 profile is active: "dev"
2022-19-07 01:45:27[INFO][restartedMain][o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor:255] Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2022-19-07 01:45:27[INFO][restartedMain][o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor:255] For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2022-19-07 01:45:27[INFO][restartedMain][o.s.d.r.c.RepositoryConfigurationDelegate:132] Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2022-19-07 01:45:27[INFO][restartedMain][o.s.d.r.c.RepositoryConfigurationDelegate:201] Finished Spring Data repository scanning in 52 ms. Found 1 JPA repository interfaces.
2022-19-07 01:45:28[INFO][restartedMain][o.s.b.w.e.t.TomcatWebServer:108] Tomcat initialized with port(s): 8080 (http)
2022-19-07 01:45:28[INFO][restartedMain][o.a.c.c.StandardService:173] Starting service [Tomcat]
2022-19-07 01:45:28[INFO][restartedMain][o.a.c.c.StandardEngine:173] Starting Servlet engine: [Apache Tomcat/9.0.64]
2022-19-07 01:45:28[INFO][restartedMain][o.a.c.c.C.[.[.[/]:173] Initializing Spring embedded WebApplicationContext
2022-19-07 01:45:28[INFO][restartedMain][o.s.b.w.s.c.ServletWebServerApplicationContext:292] Root WebApplicationContext: initialization completed in 1462 ms
2022-19-07 01:45:28[INFO][restartedMain][o.h.j.i.u.LogHelper:31] HHH000204: Processing PersistenceUnitInfo [name: default]
2022-19-07 01:45:28[INFO][restartedMain][o.h.Version:44] HHH000412: Hibernate ORM core version 5.6.9.Final
2022-19-07 01:45:28[INFO][restartedMain][o.h.a.c.Version:56] HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-19-07 01:45:28[INFO][restartedMain][c.z.h.HikariDataSource:110] HikariPool-1 - Starting...
2022-19-07 01:45:29[INFO][restartedMain][c.z.h.HikariDataSource:123] HikariPool-1 - Start completed.
2022-19-07 01:45:29[INFO][restartedMain][o.h.d.Dialect:175] HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-19-07 01:45:29[INFO][restartedMain][o.h.e.t.j.p.i.JtaPlatformInitiator:52] HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-19-07 01:45:29[INFO][restartedMain][o.s.o.j.LocalContainerEntityManagerFactoryBean:437] Initialized JPA EntityManagerFactory for persistence unit 'default'
2022-19-07 01:45:30[WARN][restartedMain][o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration:219] spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2022-19-07 01:45:30[INFO][restartedMain][o.s.b.d.a.OptionalLiveReloadServer:58] LiveReload server is running on port 35729
2022-19-07 01:45:30[INFO][restartedMain][o.s.b.w.e.t.TomcatWebServer:220] Tomcat started on port(s): 8080 (http) with context path ''
2022-19-07 01:45:30[INFO][restartedMain][c.b.a.AddressBookAppApplication:61] Started AddressBookAppApplication in 3.868 seconds (JVM running for 4.669)
2022-19-07 01:45:30[INFO][restartedMain][c.b.a.AddressBookAppApplication:12] Welcome to Address Book App!
2022-19-07 01:45:42[INFO][http-nio-8080-exec-1][o.a.c.c.C.[.[.[/]:173] Initializing Spring DispatcherServlet 'dispatcherServlet'
2022-19-07 01:45:42[INFO][http-nio-8080-exec-1][o.s.w.s.DispatcherServlet:525] Initializing Servlet 'dispatcherServlet'
2022-19-07 01:45:42[INFO][http-nio-8080-exec-1][o.s.w.s.DispatcherServlet:547] Completed initialization in 0 ms
2022-19-07 01:45:43[WARN][http-nio-8080-exec-1][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [com.bridgelabz.addressbookapp.exceptions.AddressBookException: Address Book with id 1 not found!]
2022-19-07 01:48:15[WARN][http-nio-8080-exec-7][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.addressbookapp.dto.ResponseDTO> com.bridgelabz.addressbookapp.controller.AddressBookController.createAddressBook(com.bridgelabz.addressbookapp.dto.AddressBookDTO): [Field error in object 'addressBookDTO' on field 'lastName': rejected value [kanakhara]; codes [Pattern.addressBookDTO.lastName,Pattern.lastName,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [addressBookDTO.lastName,lastName]; arguments []; default message [lastName],[Ljavax.validation.constraints.Pattern$Flag;@4cd35d24,^[A-Z][a-zA-Z]{2,}$]; default message [Last name invalid]] ]
2022-19-07 01:49:15[WARN][http-nio-8080-exec-1][o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver:208] Resolved [com.bridgelabz.addressbookapp.exceptions.AddressBookException: Address Book with id 3 not found!]
